opsdroid/database/redis/tests/test_redis.py

```diff
@@ -0,0 +1,127 @@
+import asyncio
+import json
+import logging
+from contextlib import suppress
+
+import pytest
+
+from opsdroid.database.redis import RedisDatabase
+from opsdroid.helper import JSONEncoder
+
+
+def return_async_value(val):
+    f = asyncio.Future()
+    f.set_result(val)
+    return f
+
+
+def test_init(caplog):
+    caplog.set_level(logging.DEBUG)
+    database = RedisDatabase({})
+    assert None is database.client
+    assert 0 == database.database
+    assert "localhost" == database.host
+    assert 6379 == database.port
+    assert None is database.password
+
+    log_iterator = iter(caplog.records)
+    first_log = next(log_iterator)
+    assert first_log.levelname == "DEBUG"
```

Comment:
Asserting that the first log line is a debug level message isn't really testing much of value here, and it going to be brittle. (I appreciate this is a direct port, but the tests were poor to begin with and it would be good to improve them while we convert).

It would be better to check what the line says and asset that in the whole text.

```python
assert "Loaded Redis database connector." in caplog.text()
```

Tests should check that things functioned correctly (picked up the correct default config) and that the user experience is as expected (logs a debug message to say it was loaded).

Reply:
@jacobtomlinson Thanks for the review. Although I would like to ask if I implemented your suggestion, wouldn't test fail if someone modifies those log messages? Is that situation ok?

Reply:
I'm happy with that situation. You could make the assertion simpler and just look for `Loaded Redis` or something to make it more robust.